### PR TITLE
Added Sphinx documentation for Zoomus Client

### LINKED ISSUE #149: No Documentation for Usage
Dear @prschmid ,

Please add some documentation for all functions usage and parameters.
Currently in Readme it says
`client.user.create(...)`
I am sure, one can go through the source code and understand the parameters.
But for a user, it must be simple to use in my sense without going through the source code.
Frankly speaking, those three dots doesn't help at all.

Please add some documentation in Readme itself.

### PR DESCRIPTION
**Pull request by rakeshgunduka**
Closes #149

### CODE REVIEW

**Comment by prschmid** [comment]
Thank you for this PR! I'll take a look at it along with trying to get it to work with ReadTheDocs so that we can have this library nicely documented as per the many requests!

**Comment by rakeshgunduka** [comment]
@prschmid Please review

**Comment by prschmid** [comment]
@rakesh-pankhania Apologies for the delay! I've had some urgent things come up at work that have taken up all of my attention. I promise that I will get to this shortly. Thank you so much for your effort and your patience!

File: docs/source/conf.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Zoomus'
+copyright = '2018, Patrick Schmid'
+author = 'Patrick Schmid'
+
+# The full version, including alpha/beta/rc tags
+release = '2018'
```
Comment (prschmid) [inline docs/source/conf.py:25]: Can you change this to `1.1.5`

File: docs/source/index.rst [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+*************************
+
+*Refer Create User* `https://marketplace.zoom.us/docs/api-reference/zoom-api/users/usercreate`
+
+::
+
+  params = {
+    "action": "create",
+    "user_info": {
+      "email": "dhjdfkghdskjf@fgkjfdlgjfkd.gh",
```
Comment (prschmid) [inline docs/source/index.rst:99]: Can you make this a more appealing email? E.g. "test@test.com"

File: docs/source/index.rst [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+      "type": 1,
+      "first_name": "Terry",
+      "last_name": "Jones"
```
Comment (prschmid) [inline docs/source/index.rst:102]: Nice 😆

**Review by prschmid — CHANGES_REQUESTED** [review]
Two very _very_ minor changes, but once those are updated, I'll get this merged in right away. Thank you for your time and patience!!!!

P.s. Just noticed that I mentioned the wrong person in the comment last week.

File: docs/source/conf.py (hunk 1 above)
Comment (rakeshgunduka) [reply docs/source/conf.py]: Updated to 1.1.5

File: docs/source/index.rst (hunk 2 above)
Comment (rakeshgunduka) [reply docs/source/index.rst]: Update to example@test.com

**Comment by rakeshgunduka** [comment]
@prschmid I have updated the code. Please review and close the request.

**Comment by prschmid** [comment]
@rakeshgunduka Thanks for your work on this and I appreciate your patience. With this as the starting point, we'll be able to set up something like readthedocs and make this much easier for everyone.

**Pull request merged**